src/commands/orgs/lib/org-users-helper.ts

```diff
@@ -0,0 +1,25 @@
+import { MobileCenterClient, models, clientRequest, ClientResponse } from "../../../util/apis";
+import { failure, ErrorCodes } from "../../../util/commandline";
+import { inspect } from "util";
+
+export async function getOrgUsers(client: MobileCenterClient, organization: string, debug: Function): Promise<models.OrganizationUserResponse[]> {
+  try {
+    const httpResponse = await clientRequest<models.OrganizationUserResponse[]>((cb) => client.users.listForOrg(organization, cb));
+    if (httpResponse.response.statusCode < 400) {
```

Comment:
I've seen this pattern repeated a lot - do we need to move this logic somehow into clientRequest?